src/node/Launcher.ts

```diff
@@ -813,22 +809,30 @@ function resolveExecutablePath(launcher: ChromeLauncher | FirefoxLauncher): {
         : undefined;
       return { executablePath, missingText };
     }
+    const ubuntuChromiumPath = '/usr/bin/chromium-browser';
+    if (
+      product === 'chrome' &&
+      os.platform() !== 'darwin' &&
```

Comment:
It was tempting to change this to `os.platform() === 'linux'` because `/usr/bin/chromium-browser` seemed only applicable to Ubuntu from what I could tell.

Reply:
It didn't seem like `os.arch() === 'arm64'` should be retained anymore. Whether `chromium-browser` amd64 or arm64 build is installed it will be in the same location. Should it be retained so that it only behaves this way on arm64 systems because that's the way it used to behave? The current conditions here would add preference to a pre-installed `/usr/bin/chromium-browser` instead of downloading one on non-darwin platforms.